source/Calamari/Kubernetes/Integration/CaptureCommandOutput.cs

```diff
@@ -8,14 +9,18 @@ public interface ICommandOutput
     {
         Message[] Messages { get; }
         IEnumerable<string> InfoLogs { get; }
+        string MergeInfoLogs();
     }
+
     public class CaptureCommandOutput : ICommandInvocationOutputSink, ICommandOutput
     {
         private readonly List<Message> messages = new List<Message>();
         public Message[] Messages => messages.ToArray();
 
         public IEnumerable<string> InfoLogs => Messages.Where(m => m.Level == Level.Info).Select(m => m.Text).ToArray();
 
+        public string MergeInfoLogs() => string.Join(Environment.NewLine, InfoLogs);
```

Comment:
What are your thoughts on having the newline character depend on the OS where Calamari is running?

Reply:
I don't have a lot of them - there are existing examples of joining these log messages with `Environment.NewLine` so my assumption is that it works fine in all cases we're using it at the moment.

Sticking with Unix newlines might be a good idea to be consistent - perhaps there is a bug we just haven't seen come up yet?
By the same token, perhaps there is no bug because windows people tend to have everything on windows so we're just getting lucky...

I don't have any good answers aside from sticking with the status quo

Reply:
cool, status quo is good